overlord/devicestate/devicemgr.go

```diff
@@ -2201,6 +2201,31 @@ func (m *DeviceManager) systems() ([]*System, error) {
 	return systems, nil
 }
 
+func snapdVersionByTypeFromSeed20(s seed.Seed, types []snap.Type) (systemSnapdVersions *install.SystemSnapdVersions, err error) {
+	perf := &timings.Timings{}
+	if err := s.LoadEssentialMeta(types, perf); err != nil {
```

Comment:
can't we avoid this, by adding TypeSnapd to the system load calls in the couple of callers to this?